pontoon/base/views.py

```diff
@@ -440,9 +444,13 @@ def entities(request):
     except (MultiValueDictKeyError, ValueError) as err:
         return HttpResponseBadRequest('Bad Request: {error}'.format(error=err))
 
-    project = get_object_or_404(Project, slug=project)
     locale = get_object_or_404(Locale, code__iexact=locale)
 
+    if project == 'all-projects':
+        project = Project(slug=project)
```

Comment:
Yeah, I was thinking about creating a fake `all-projects` instance of the `Project` model, too. 

I'd also like to get rid of the `if project == all-projects` checks if possible.

And, make sure this actually performs in production.